src/dotnet/APIView/APIViewWeb/Controllers/PullRequestController.cs

```diff
@@ -33,14 +33,15 @@ public PullRequestController(PullRequestManager pullRequestManager, ILogger<Auto
             string packageName,
             int pullRequestNumber = 0,
             string codeFile = null,
-            string baselineCodeFile = null)
+            string baselineCodeFile = null,
+            bool commentOnPR = true)

```

Comment:
Once we start to use this for SDK automation scenario we will also need to make things like pullRequestNumber optional because it might not exist for the language repo.

Reply:
Sure. But SDK automation will have PR for swagger PR. right?

Reply:
Yes but the SDK automation will run independent for each language. We also need to make sure we can support if this isn't triggered by any PR.